slides/src/content/introduction.md

```diff
@@ -37,4 +53,247 @@ FIXME
   - Supporting projects: CLI, Universal, Nativescript
 - A complete and opinionated solution
 - You don't need to be a Javascript expert
-- Can be integrated with Redux
\ No newline at end of file
+- Can be integrated with Redux
+
+
+---
+
+## Getting Set Up
+
+1. Install NVM (Node Version Manager) -- [nvm.sh](http://nvm.sh)
+1. Install the latest LTS version of node -- `nvm install --lts`
+1. Install Angular CLI -- `npm install --global @angular/cli`
+
+---
+
+## Creating a New Project
+
+`ng create [project-name]`
+
+This will:
+
+- Install Angular
+- Install node dependencies
+- Create scaffolding
+
+---
+
+## Project Structure
+
+- `package.json` -- project metadata
+- `node_modules` -- libraries and dependencies used by our application
+- `src` folder -- application code
+- `tsconfig.json` -- TypeScript configuration
+
+---
+
+## TypeScript
+
+- TypeScript is exactly like JavaScript, but with types
+- Has ES6 and ES7 features
+- Compiles to JavaScript before being run in the browser
+- No runtime support for types
+
+---
+
+## First Look at the Component
+
+```typescript
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: '<h1>{{title}}</h1>',
+  styles: ['h1 { color: font-size: 30px; }']
+})
+export class AppComponent {
+  title = 'app works!';
+}
+```
+
+- Import statement gives you access to Angular components
+- `@Component` is a decorator function that tells Angular this class is a component
+- `selector` is the way you intend to reference the component. Used like an HTML tag `<app-root></app-root>`
+- `template` is what gets rendered onto the page. For web applications, this is HTML
+- `styles` are css stylesheets that get applied to your component
+- `AppComponent` is a class that decides how the component should act
+
+---
+
+## What is a Component?
+
+- The core building block of an Angular application is the component
+- In Angular, a component is some chunk of application logic combined with something that gets displayed
+- Everything being displayed on any page should be associated with some component
+
+---
+
+## Component Template Syntax
+
+- `<h1>{{title}}</h1>` represents __interpolation__
+- Interpolation attempts to evaluate the expression between the curly braces and turn that into a string
+- Symbols (such as title) are evaluated within an expression context, in this case, the component's class members
+
+---
+
+## Generating a New Component
+
+- We'll use Angular CLI to create a new component
+- `ng generate component [component path]`
+- Component gets added to the list of declarations
+- Generated component follows the same structure as `AppComponent`
+- Prefix `app` on both components. This is an Angular best practice
```

Comment:
In the introduction we are not asking the user to follow along to create a sample app, nor are we creating one. Do you still think its worth to include a tree?